# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/nsMacFinderProgress.h
# Commit: 5f48ef706159
# Full Hash: 5f48ef706159275c966c80b13cf09d7dcc1db932
# Author: Christoph Walcher <christoph-wa@gmx.de>
# Date: 2019-06-15 21:44:24
# Regressor Bug: 909760
# File Overlap Count: 1
# Description:
#   Bug 909760 - Show download progress in the MacOS Finder r=spohl,mak
#   
#   Show download progress in the MacOS Finder
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30688
# ==============================================================================

diff -r fc8f53ac1fa2 -r 5f48ef706159 widget/cocoa/nsMacFinderProgress.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/widget/cocoa/nsMacFinderProgress.h	Sat Jun 15 15:46:36 2019 +0000
@@ -0,0 +1,24 @@
+/* -*- Mode: c++; tab-width: 2; indent-tabs-mode: nil; -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#ifndef _MACFINDERPROGRESS_H_
+#define _MACFINDERPROGRESS_H_
+
+#include "nsIMacFinderProgress.h"
+#include "nsCOMPtr.h"
+
+class nsMacFinderProgress : public nsIMacFinderProgress {
+ public:
+  nsMacFinderProgress();
+
+  NS_DECL_ISUPPORTS
+  NS_DECL_NSIMACFINDERPROGRESS
+
+ protected:
+  virtual ~nsMacFinderProgress();
+
+  NSProgress* mProgress;
+};
+
+#endif